src/libraries/System.Net.Security/src/System/Net/Security/SslCertificateTrust.cs

```diff
@@ -43,10 +42,18 @@ public static SslCertificateTrust CreateForX509Store(X509Store store, bool sendT
         [UnsupportedOSPlatform("windows")]
```

Comment:
I'm wondering if there is better way how to do this. It is unsupported on Windows only if `sendTrustInHandshake` is true. cc: @bartonjs

Reply:
I... believe... that if it's conditional based on inputs that we just don't annotate it.  @jeffhandley / @buyaa-n ?

Reply:
> if it's conditional based on inputs that we just don't annotate it

That is right